.github/workflows/format.yml

```diff
@@ -1,23 +1,14 @@
-name: Persubmit Checks
+name: Code Format Check
 on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-    paths:
-      - '**.h'
-      - '**.cpp'
-      - '**.py'
-      - '**.rst'
-      - '**.md'
-      - '!examples/**.py'
+  #pull_request:
```

Comment:
Is this intentional?

Reply:
Yes, to help debug with another yml. Will clean before everything works and merge.